src/snowflake/snowpark/mock/_functions.py

```diff
@@ -1085,6 +1085,43 @@ def mock_to_variant(expr: ColumnEmulator):
     return res
 
 
+def _object_construct(exprs, drop_nulls):
+    import pandas
```

Comment:
Just curious, why not put import at the beginning? Is it a limitation of mock?

Reply:
The regular execution mode of snowpark-pandas doesn't require pandas, but local testing does. IIRC I tried importing at the module level, but that broke some of the non-pandas tests.

Reply:
I see, got it! Ideally I think we should do all the import at module level to make test code more clean, but we can do that in the future